Respect --disable-nls by using gettext's POSUB for the po subdirectory
--- ../Makefile.am	2026-09-23 21:06:16.155694056 +0300
+++ ../Makefile.am	2026-09-23 21:06:31.676261053 +0300
@@ -1,7 +1,7 @@
 ## Process this file with automake to create Makefile.in.
 AUTOMAKE_OPTIONS = 1.14
 
-SUBDIRS = po lib src doc contrib misc
+SUBDIRS = $(POSUB) lib src doc contrib misc
 
 if HAVE_TESTS
     SUBDIRS += tests
